### DISCUSSION TITLE
How to disable this (see screenshot) when I horizontal overscroll the TouchPad？

### DISCUSSION THREAD

**Discussion by linesoft2** [tauri-apps/tauri#5958] [Q&A] [answered]
![QQ截图20230102151152](https://user-images.githubusercontent.com/25446947/210203005-594b4b66-7c27-40ba-bb2b-e3fa45556fba.png)

**Comment by FabianLars** [comment, marked as answer]
Can you try adding this?
add these dependencies to your cargo.toml file:
```toml
[dependencies]
webview2-com = "0.19"
windows = "0.39"

```
and then add a .setup hook to the builder in your main.rs file similar to this
```rs
use tauri::Manager;
use webview2_com::Microsoft::Web::WebView2::Win32::ICoreWebView2Settings6;
use windows::core::Interface;

fn main() {
    tauri::Builder::default()
        .setup(|app| {
            let window = app.get_window("main").unwrap();
            window.with_webview(|webview| unsafe {
                let settings = webview.controller().CoreWebView2().unwrap().Settings().unwrap();
                let settings = settings.cast::<ICoreWebView2Settings6>().unwrap();
                let settings.SetIsSwipeNavigationEnabled(false).unwrap();
            }).unwrap();
            Ok(())
        })
        .invoke_handler(tauri::generate_handler![])
        .run(tauri::generate_context!())
        .expect("error while running tauri application");
}
```

This was the only relevant setting i found but i don't have a trackpad or touchscreen available to test it myself right now

**Reply by linesoft2** [reply]
Thank you. It solves my problem, but I think you should use it as the default setting, like Electron.

**Reply by FabianLars** [reply]
I agree

**Comment by mad-jose** [comment]
although the arrow mark is gone, so as to disable navigation, it still shows a white bar when swiping

**Reply by IsaacMunyoz** [reply]
Same problem

**Reply by IsaacMunyoz** [reply]
html,body {
   overscroll-behavior: none;
}
I solved with a css config

**Reply by ngryman** [reply]
This didn't work for me.

**Comment by ngryman** [comment]
There's actually a simple cross-platform CSS solution to this:

```css
body {
  overflow: hidden;
  position: fixed;
  width: 100vw;
  height: 100vh;
}
```

Then put `overflow: auto` to the containers you want to scroll.

**Comment by kurdin** [comment]
Yeah, just use css, not need to any extra packages.
